File: app/src/main/java/com/liyafeng/videofactory/adapter/MainAdapter.java
package com.liyafeng.videofactory.adapter;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;
import com.liyafeng.module_convert.ui.MainConvertFragment;

public class MainAdapter extends FragmentPagerAdapter {

    public MainAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int i) {
        if (i == 0) {
            return MainConvertFragment.newInstance();
        } else {
            return MainConvertFragment.newInstance();
        }
    }

    @Override
    public int getCount() {
        return 2;
    }
}

File: app/src/main/java/com/liyafeng/videofactory/adapter/MainAdapter.kt
package com.liyafeng.videofactory.adapter

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter
import com.liyafeng.module_convert.ui.MainConvertFragment
import com.liyafeng.module_mine.ui.MineFragment

class MainAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    override fun getItem(i: Int): Fragment {
        return if (i == 0) {
            MainConvertFragment.newInstance()
        } else {
            MineFragment()
        }
    }

    override fun getCount(): Int {
        return 2
    }
}
